client/cli/src/main.rs

```diff
@@ -387,14 +415,19 @@ enum Command {
     EpochCount,
 }
 
-trait ErrorHandler<T, E> {
+/// Extension trait for values that either return `T` or terminate the program
+/// with CLI error
+trait OrTerminate<T> {
     fn or_terminate(self, err: CliErrorKind, msg: &str) -> T;
+}
+
+trait ErrorHandler<T, E>: OrTerminate<T> {
```

Comment:
I kind of don't understand the meaning of this method, there is no comments explaining the abstraction that it's trying to create, so I moved the part that I can make sense of, and left `transform` as was.

Reply:
Seems to be from @your-sudden-death, maybe he can elaborate.